src/main.rs

```diff
@@ -33,7 +33,12 @@ enum Command {
     /// Generate a new account for the initialized HD wallet.
     New { path: Option<String> },
     /// Initialize the HD wallet. If it is already initialized this will remove the old one.
-    Init { path: Option<String> },
+    Init {
+        #[clap(long)]
+        phrase: Option<String>,
```

Comment:
I wonder if eventually we should accept this phrase via a password type prompt rather than command-line input to avoid having this end up in a user's shell/bash history? I haven't used another wallet CLI just yet so not sure what standard practice is here. Perhaps we can leave it for now and open an issue to follow-up.

Reply:
Yes.